src/mmu/gpio.rs

```diff
@@ -0,0 +1,30 @@
+use std::ops::{Index, IndexMut};
+
+const GPIO_SIZE: usize = 0x80;
+
+#[derive(Debug)]
+pub struct GPIO {
+    data: Vec<u8>,
```

Comment:
Since the size should not be changed, a Box<[u8]> might be more appropriate

Reply:
If you make the change to a Box, add .into_boxed_slice() after the vec![] in the new method

Reply:
8e157c8